core/src/test/scala/dx/core/languages/wdl/InstanceTypesTest.scala

```diff
@@ -304,6 +299,10 @@ class InstanceTypesTest extends AnyFlatSpec with Matchers {
                           os = Some(Constants.DefaultExecutionEnvironment))
   }
 
+  it should "create a runtime with default instance type" in {
```

Comment:
Created a separate test case for convenience. 
Also resolve to empty `InstanceTypeRequest` if no instance override is happening, which is ultimately resolved to `InstanceTypeDB.default`